operations/utility_storage/Identify_Ceph_Latency_Issues.md

```diff
@@ -18,7 +18,7 @@ This procedure requires admin privileges.
 
     Example output:
 
-    ```
+    ```bash
```

Comment:
this should probably be `text`, not `bash`.  I assume you changed this because the check was complaining?

Reply:
I did add that because the check was failing and I have switched it to 'text'. Thanks!